README.md

```diff
@@ -130,24 +137,27 @@ cd Unvanquished/
 Produced files will be stored in a new directory named `build`.
 
 ```sh
-cmake -H. -Bbuild
-cmake --build build -- -j4
+mkdir build; cd build
+cmake ..
+make -j4
 ```
 
 ### MSYS2
 
 ```sh
-cmake -H. -Bbuild -G "MSYS Makefiles"
-cmake --build build -- -j4
+mkdir build; cd build
+cmake .. -G "MSYS Makefiles"
+make -j4
 ```
 
 ### Linux cross-compile to Windows
 
 💡️ For a 32-bit build use the `cross-toolchain-mingw32.cmake` toolchain file instead.
 
 ```sh
-cmake -H. -Bbuild -DCMAKE_TOOLCHAIN_FILE=cmake/cross-toolchain-mingw64.cmake
-cmake --build build -- -j4
+mkdir build-win64; cd build-win64
```

Comment:
Isn't it better to do?

```sh
mkdir -p build/win64; cd build/win64
cmake ../.. -DCMAKE_TOOLCHAIN_FILE=cmake/cross-toolchain-mingw64.cmake
…
```

Maybe personal taste, but I'm still surprised when I find people not knowing about it. I'm doing this since probably 10 years, so [I don't know what was annoying about ignoring only the `build/` folder](https://github.com/Unvanquished/Unvanquished/pull/2326).

Reply:
Actually the path to the toolchain is wrong anyway, it is either:

```sh
mkdir -p build/win64; cd build/win64
cmake ../.. -DCMAKE_TOOLCHAIN_FILE=../../daemon/cmake/cross-toolchain-mingw64.cmake
```

or:

```sh
mkdir -p build-win64; cd build-win64
cmake .. -DCMAKE_TOOLCHAIN_FILE=../daemon/cmake/cross-toolchain-mingw64.cmake
```

or:

```sh
cmake -H. -Bbuild/win64 -DCMAKE_TOOLCHAIN_FILE=daemon/cmake/cross-toolchain-mingw64.cmake
```

or:

```sh
cmake -H. -Bbuild-win64 -DCMAKE_TOOLCHAIN_FILE=daemon/cmake/cross-toolchain-mingw64.cmake
```

Reply:
Fixed the toolchain path.

Yeah that's how I have it set up on my machine - all the build directories are subdirectories of `build`. Just keeping it simple for the README. Also if in other parts of the README we suggest that `build` itself is used, I don't want to make suggestions that would have one build dir inside another if used together.